components/list/list-item-content.js

```diff
@@ -5,8 +5,10 @@ class ListItemContent extends LitElement {
 	static get styles() {
 		return [ bodySmallStyles, bodyCompactStyles, css`
 			.d2l-list-item-content-text {
+				color: var(--d2l-list-item-content-text-color, null);
```

Comment:
`null` doesn't seem right... do you want to just not specify a fallback maybe?

Reply:
So I did place `null` in places when I wanted it to fail. So I could set it and unset it with `initial`.

Just removed the `null` and nothing changed so I'll remove this one :)